docs/UserGuide.md

```diff
@@ -11,7 +11,9 @@ title: User Guide
     3.1 [Command Format](#31-command-format)<br>
     3.2 [Add a patient: `addpatient`](#32-add-a-patient-addpatient)<br>
     3.3 [Edit a patient: `editpatient`](#33-edit-a-patient-editpatient)<br>
-    3.4 [Save the data](#34-save-the-data)<br>
+    3.4 [Add rooms in hotel: `addRooms`](#34-add-rooms-in-hotel-addrooms)<br>
+    3.5 [Finds the next room which is free to use: `findRoom`](#35-finds-the-next-room-which-is-free-to-use-findroom)<br>
```

Comment:
I think "next" could be a little vague, coz the user may not know which room they are in, maybe "finds the first free room" could be better?

Reply:
okay can, I will change it